oracle_gis.c

```diff
@@ -725,9 +726,14 @@ char *
 ewkbHeaderFill(oracleSession *session, ora_geometry *geom, char * dest)
 {
 	const unsigned srid = ewkbSrid(session, geom);
-	const ub1 flags = ((3 == ewkbDimension(session, geom)) ? 0x01 : 0x00 );
+	ub1 flags;
 	ub1 s[3];
 
+	if (ewkbIsMeasured(session, geom))
+		flags = 0x02;  /* two-dimensional geometry with measure */
+	else
+		flags = ((3 == ewkbDimension(session, geom)) ? 0x01 : 0x00 );
+
```

Comment:
Maybe add an error 'unsupportded'

    if ((3 == ewkbDimension(session, geom)) && ewkbIsMeasured(session, geom))

before that.